src/app/components/App.jsx

```diff
@@ -17,10 +17,14 @@ import MiniHeader from 'app/components/modules/MiniHeader';
 import tt from 'counterpart';
 import PageViewsCounter from 'app/components/elements/PageViewsCounter';
 import {serverApiRecordEvent} from 'app/utils/ServerApiClient';
-import { APP_NAME, VESTING_TOKEN, LIQUID_TOKEN } from 'app/client_config';
 import {key_utils} from 'steem/lib/auth/ecc';
 import resolveRoute from 'app/ResolveRoute';
 import {VIEW_MODE_WHISTLE} from 'shared/constants';
+import config from 'config';
+
+const {
+    LIQUID_TOKEN
```

Comment:
what happened to APP_NAME and VESTING_TOKEN?

Reply:
Not being used.

Reply:
ok